metau-capstone/View Controllers/CreateViewController.m

```diff
@@ -58,6 +58,25 @@ - (IBAction)didTapImport:(UIButton *)sender {
 }
 
 
+- (IBAction)didTapShare:(id)sender {
+    NSURL *url= [[NSURL alloc] init];
+    PFUser *const user = [PFUser currentUser];
+    // Construct URL
+    NSURLComponents *components = [[NSURLComponents alloc] init];
+    components.scheme = @"com.shirleyzhu.metau-capstone";
```

Comment:
Curious, how do the phone know how to interpret this and open the application?

Reply:
I registered "com.shirleyzhu.metau-capstone" as a URL scheme by adding the URL scheme in the Info tab of my project in Xcode. The URL scheme registration then specifies that all URLs starting with "com.shirleyzhu.metau-capstone" should be redirected to my app.

<img width="1910" alt="Screen Shot 2022-08-05 at 2 50 40 PM" src="https://user-images.githubusercontent.com/44847817/183215709-73f9bb7f-94c9-49c0-9249-81c80511bf2a.png">

I am not sure what happens more in the backend, but will let you know if I find out more!